qiskit/circuit/library/hidden_linear_function.py

```diff
@@ -52,7 +52,7 @@ class HiddenLinearFunction(QuantumCircuit):
     logarithmically with :math:`n`. Thus this circuit is an example
     of quantum advantage with shallow circuits.
 
-    **Reference Circuit:**
+    Reference Circuit:
 
         .. plot::
            :alt: Diagram illustrating the previously described circuit.
```

Comment:
Probably the `.. plot` needs to be unindented?

Reply:
Honestly, I'm actually surprised that the docs build even _works_ with the extra indentation - Sphinx is usually picky about it.